circle.yml

```diff
@@ -225,6 +225,38 @@ jobs:
            name: Test
            command: |
              test -f out/bin/ctags && ( file out/bin/ctags | grep -q 'ARM aarch64' )
+   centos6:
+     working_directory: ~/universal-ctags
+     docker:
+       - image: centos:6
+     steps:
+       - run:
+           name: Rearrange CentOS-Base.repo file
+           command: |
+             sed -i "s|#baseurl=|baseurl=|g" /etc/yum.repos.d/CentOS-Base.repo
+             sed -i "s|mirrorlist=|#mirrorlist=|g" /etc/yum.repos.d/CentOS-Base.repo
+             sed -i "s|http://mirror\.centos\.org/centos/\$releasever|https://vault\.centos\.org/6.10|g" /etc/yum.repos.d/CentOS-Base.repo
+       - run:
+           name: Install tools for buildig
+           command: |
+             yum install -y git gcc make autoconf automake pkgconfig
+       - run:
+           name: Get the source code of u-ctags
+           command: |
+             git clone https://github.com/universal-ctags/ctags
+       - run:
+           name: Build
+           command: |
+             (
+             cd ctags
+             bash ./autogen.sh
+             ./configure --enable-debugging
+             make -j 2
+             )
+       - run:
+           name: Don't run test cases because python3 is no available
```

Comment:
Red Hat has added `Python 3.4` for `CentOS 6` through the `EPEL` repository.

easiest way to install `Python 3.4` on `CentOS 6` from `EPEL` repository:

```
yum install -y epel-release
yum install -y python34
```

Reply:
Thank you.

Reply:
I found our test harness needs python 3.5.
misc/units.py uses subprocess.run(), which is introduced in python3.5.
So I will not run the test cases on centos6@circleci.